File: Practice/py/2/IfElse.py
# taking input and storing it in n as an integer
n = int(input())
# if n is odd
if n % 2 == 1:
    # print Weird
    print("Weird")
# if n is even
else:
    # if n is from 2 to 5
    if n in range(2, 6):
        # print Not Weird
        print("Not Weird")
    # if n is from 6 to 20
    elif n in range(6, 21):
        # print Weird
        print("Weird")
    # if n is more than 20
    elif n > 20:
        # print Not Weird
        print("Not Weird")

File: Practice/java/3/IfElse.java
// importing Scanner class
import java.util.Scanner;
// main class
public class IfElse {
	// main fucntion
	public static void main(String args[]) {
		// creating object of Scanner class for taking user input
		Scanner sc = new Scanner(System.in);
		// integer variable
		int n = 0;
		// scanning integer input
		n = sc.nextInt();
		// string variable
		String result = null;
		// if n is odd
		if (n % 2 == 1)
			// result is Weird
		    result = "Weird";
		// if n is even
		else if (n % 2 == 0)
			// if n is in between 2 and 5 inclusive
			if (2 <= n && n <= 5)
				// result is Not Weird
				result = "Not Weird";
			// if n is in between 6 and 20 inclusive
			else if (6 <= n && n <= 20)
				// resutl is Weird
				result = "Weird";
			// if n is greater than 20
			else if (n > 20)
				// result is Not Weird
				result = "Not Weird";
		// printing final result
		System.out.println(result);
	}
}
